homeassistant/components/music_assistant/strings.json

```diff
@@ -37,6 +37,100 @@
       "description": "Check if there are updates available for the Music Assistant Server and/or integration."
     }
   },
+  "services": {
+    "play_media": {
+      "name": "Play Media (advanced)",
```

Comment:
I am not sure how I feel about the `(advanced)` though

Reply:
It is to easy recognize it from the default/limited play media - I'm ok with dropping the advanced postfix though but not sure if that is recognizeable enough for users

Reply:
It will be prefixed with the integration title still, right?

Reply:
Yeah, I have adjusted it.